src/validator/CreditCardVendorReading.java

```diff
@@ -0,0 +1,7 @@
+package validator;
+
+import java.util.ArrayList;
+
+public interface CreditCardVendorReading {
+	public ArrayList<CreditCardVendor> returnVendorArray();
```

Comment:
returnVendors or returnVendorList fits better as method name (as this methods returns list)